src/libYARP_OS/include/yarp/os/SystemClock.h

```diff
@@ -30,6 +30,7 @@ class YARP_OS_API yarp::os::SystemClock : public Clock {
 
     static double nowSystem();
     static void delaySystem(double seconds);
+    static void yieldSystem();
```

Comment:
The `yieldSystem()` method is in my opinion wrong... there is no need for this in `SystemClock`, `yield` is not something related to `time` or to the clock, but to the execution of the thread, hence it should call `yarp::os::Thread::yield()` and `yarp::os::Time::yield()` should be deprecated...

Reply:
or even better, call the c++11 yield() function directly...

Reply:
damn, @traversaro you are fast!

Reply:
Make sense. Done in the last version of the commit.